src/components/Header.js

```diff
@@ -7,7 +7,7 @@ function Header() {
     <div className="Header">
       <nav className="navItems">
         <h1 className="logo">
-          <Link to="/">Boostore</Link>
+          <Link to="/">Boostore CMS</Link>
```

Comment:
- [ ] Please make sure that the header and form has `font-family: Montserrat;` and your font styles, weight and size should also match the provided [design guidelines ](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). See Screenshots for reference:

Navbar
---
Your Implementation | Zeplin Design 
----|----
<img width="815" alt="Screenshot 2023-09-15 at 11 56 22 AM" src="https://github.com/paulshaun-kdtk/React_Book_Store/assets/44798044/93789c07-1a9e-4509-b675-21776ba27cf5">|<img width="1003" alt="Screenshot 2023-09-15 at 11 50 21 AM" src="https://github.com/paulshaun-kdtk/React_Book_Store/assets/44798044/f71b81cd-9671-4309-8122-d845777a45d5">

Things to modify:

- [ ] logo font-family, font-style, and font-size
- [ ] Navlink's font color, style, weight, size
- [ ] Missing profile icon on the right mot corner of the navbar 

Form
---
Your Implementation | Zeplin Design 
----|----
<img width="960" alt="Screenshot 2023-09-15 at 11 57 44 AM" src="https://github.com/paulshaun-kdtk/React_Book_Store/assets/44798044/155cd971-ca5f-4f00-9282-28c3b162d32b">
|<img width="936" alt="Screenshot 2023-09-15 at 11 57 58 AM" src="https://github.com/paulshaun-kdtk/React_Book_Store/assets/44798044/b9db4fc3-1744-4409-b129-3ca9118f370c">

Things to modify:

- [ ] `Add new book` font-family, font-weight and font-size
- [ ] Placeholder color and font-family
- [ ] Size and padding of the input fields and `add book` button
- [ ] Missing divider line above the form